### PR TITLE
CLDR-15956 Specify relations between the unit preferences data and the locale identifier

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-15956

- [ ] This PR completes the ticket.

### CODE REVIEW

File: docs/ldml/tr35-info.md [hunk 1]
```diff
@@ -252,8 +252,11 @@ The default preference for several locale items is based solely on a [unicode_re
 * Default measurement system and paper size (see [Measurement System Data](tr35-general.md#Measurement_System_Data))
 * Default units for specific usage (see [Preferred Units for Specific Usages](#Preferred_Units_For_Usage), below)
 
+The mu, ms, and rg keys also interact with the base locale and the unit preferences. For more information, see _Section 14 [Unit Preferences](#Unit_Preferences)._
+
 #### 2.4.1 <a name="Preferred_Units_For_Usage" href="#Preferred_Units_For_Usage">Preferred Units for Specific Usages</a>
 
+The determination of preferred units depends on the locale identifer: the keys ms, mu, rg, the base locale (language, script, region) and the user preferences.
```
Comment (sffc) [inline docs/ldml/tr35-info.md:259]:
```suggestion
The determination of preferred units depends on the locale identifer: the keys mu, ms, rg, the base locale (language, script, region) and the user preferences.
```

File: docs/ldml/tr35-info.md [hunk 2]
```diff
@@ -1060,6 +1063,46 @@ The [unitsTest.txt](https://github.com/unicode-org/cldr/blob/main/common/testDat
 
 Different locales have different preferences for which unit or combination of units is used for a particular usage, such as measuring a person’s height. This is more fine-grained than merely a preference for metric versus US or UK measurement systems. For example, one locale may use meters alone, while another may use centimeters alone or a combination of meters and centimeters; a third may use inches alone, or (informally) a combination of feet and inches.
 
+### 14.2 <a name="Unit_Preferences_Data" href="#Unit_Preferences_Data">Unit Preferences Overrides</a>
+
+The determination of preferred units depends on the locale identifer: the keys ms, mu, rg, their values, the base locale (language, script, region) and the user preferences data.
```
Comment (sffc) [inline docs/ldml/tr35-info.md:1068]:
```suggestion
The determination of preferred units depends on the locale identifer: the keys mu, ms, rg, their values, the base locale (language, script, region) and the user preferences data.
```

File: docs/ldml/tr35-info.md [hunk 3]
```diff
@@ -1060,6 +1063,46 @@ The [unitsTest.txt](https://github.com/unicode-org/cldr/blob/main/common/testDat
...
+
+The strongest is the mu key, then the ms key, then the rg key. Beyond that the region of the locale identifer is used, and if not present, the likely-subtag region. For example:
+
+|   | Locale                                | Result     | Comment                                                            |
+|---|---------------------------------------|------------|--------------------------------------------------------------------|
+| 1 | en-u-rg-uszzzz-ms-ussystem-mu-celsius | Celsius    | despite the rg and ms settings for US, and the likely region of US |
+| 2 | en-u-rg-uszzzz-ms-metric              | Celsius    | despite the rg setting for US, and the likely region of US         |
+| 3 | en-u-rg-dezzzz will use celsius       | Celsius    | despite the likely region of US                                    |
```
Comment (sffc) [inline docs/ldml/tr35-info.md:1076]:
```suggestion
| 3 | en-u-rg-dezzzz                        | Celsius    | despite the likely region of US                                    |
```

File: docs/ldml/tr35.md [hunk 4]
```diff
@@ -3663,6 +3669,8 @@ Proposed Update for CLDR Version 42.
     * Described the new `dateTimeFormat type="atTime"` pattern and when to use it versus the standard `dateTimeFormat` pattern.
 * [Person Names](tr35-personNames.md#Contents)
     * Added a new Part 8, Person Names.
+* [Unit Preferences](tr35-info.md#Unit_Preferences)
+   * Added a new subsection to  specify the interaction of the unit Preferences data with the locale keys ms, mu, and rg, and the base locale.
```
Comment (sffc) [inline docs/ldml/tr35.md:3673]:
```suggestion
   * Added a new subsection to  specify the interaction of the unit Preferences data with the locale keys mu, ms, and rg, and the base locale.
```

**Review by sffc — APPROVED** [review]

**Review by pedberg-icu — APPROVED** [review]

**Comment by AEApple** [comment]
@macchiati - Don't you need a final approver before you merge?

@srl295 - are we missing any checks on the maint-42 branch?

**Comment by srl295** [comment]
> @macchiati - Don't you need a final approver before you merge?
> 
> 
> 
> @srl295 - are we missing any checks on the maint-42 branch?

Peter had approved

**Comment by AEApple** [comment]
@srl295 - Yes, but then it was not approved anymore after Mark made the fixes for Shane's comments? I'm still confused about this one. I thought we had to have an approval that was active before the final merge?

**Comment by srl295** [comment]
> @srl295 - Yes, but then it was not approved anymore after Mark made the fixes for Shane's comments? I'm still confused about this one. I thought we had to have an approval that was active before the final merge?

@AEApple @macchiati  there was this checkbox NOT checked for maint-* branches. I've now checked it. I think it may be a newer setting.

- [X] Dismiss stale pull request approvals when new commits are pushed

**Comment by macchiati** [comment]
I didn't look to see whether another review had been done, just whether the
button was enabled. Looks like there wasn't a rereview, but it this
particular case that wasn't an issue.

Glad that you found and fixed.

**Comment by srl295** [comment]
Should be fixed now

**Pull request merged**